packages/viewer/src/components/viewer-measurement-precise/viewer-measurement-precise.tsx

```diff
@@ -23,7 +23,7 @@ export class ViewerMeasurementPrecise {
   @Prop({ mutable: true })
   public measurementController?: MeasurementController;
 
-  @Prop()
+  @Prop({ mutable: true })
```

Comment:
This only affects changes made internally by the component. Consumers can still set this property and Stencil will rerender, invoke callbacks, etc.

Reply:
Are you suggesting I should look at the where this is being used instead of changing something here?

Reply:
Ya, I think the issue is elsewhere. Will ping ya.